build.sbt

```diff
@@ -14,56 +13,38 @@ lazy val core = project
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.EagerSlickDatabase"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.SlickDriver$"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.SlickExtension"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.LazySlickDatabase"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.SlickExtensionImpl"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.SlickDriver"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.EagerSlickDatabase$"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.SlickExtension$"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.persistence.jdbc.util.DefaultSlickDatabaseProvider"),
-        ProblemFilters.exclude[IncompatibleResultTypeProblem](
-          "akka.persistence.jdbc.journal.JdbcAsyncWriteJournal.slickDb"),
-        ProblemFilters.exclude[IncompatibleResultTypeProblem](
-          "akka.persistence.jdbc.snapshot.JdbcSnapshotStore.slickDb")),
-    // special handling as we change organization id
-    mimaPreviousArtifacts := ProjectAutoPlugin.determineMimaPreviousArtifacts(scalaBinaryVersion.value))
+    mimaBinaryIssueFilters ++= Seq()
+  )
 
 lazy val migration = project
   .in(file("migration"))
   .enablePlugins(Publish)
   .disablePlugins(SitePlugin, MimaPlugin)
   .settings(
-    name := "akka-persistence-jdbc-migration",
+    name := "akka-persistence-postgres-migration",
     libraryDependencies ++= Dependencies.Migration,
     publish / skip := true)
 
 lazy val docs = project
   .enablePlugins(ProjectAutoPlugin, AkkaParadoxPlugin, ParadoxSitePlugin, PreprocessPlugin, PublishRsyncPlugin)
   .disablePlugins(MimaPlugin)
   .settings(
-    name := "Akka Persistence JDBC",
+    name := "Akka Persistence Postgres",
     publish / skip := true,
     makeSite := makeSite.dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
     previewPath := (Paradox / siteSubdirName).value,
-    Preprocess / siteSubdirName := s"api/akka-persistence-jdbc/${if (isSnapshot.value) "snapshot"
+    Preprocess / siteSubdirName := s"api/akka-persistence-postgres/${if (isSnapshot.value) "snapshot"
       else version.value}",
     Preprocess / sourceDirectory := (LocalRootProject / ScalaUnidoc / unidoc / target).value,
-    Paradox / siteSubdirName := s"docs/akka-persistence-jdbc/${if (isSnapshot.value) "snapshot" else version.value}",
+    Paradox / siteSubdirName := s"docs/akka-persistence-postgres/${if (isSnapshot.value) "snapshot" else version.value}",
     Compile / paradoxProperties ++= Map(
         "project.url" -> "https://doc.akka.io/docs/akka-persistence-jdbc/current/",
         "canonical.base_url" -> "https://doc.akka.io/docs/akka-persistence-jdbc/current",
```

Comment:
jdbc

Reply:
This is a url to the Akka documentation page. We will eventually replace it with our github page. Also I'm going to get rid off Paradox.